msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -758,14 +774,85 @@ The `publisherDocsUrl` is a HTTPS URL to an information page for admins to get g
 
 Specifies the SaaS offer associated with your app.
 
-|Name| Type| Maximum size | Required | Description|
+|Name| Type|Maximum size|Required|Description|
 |---|---|---|---|---|
 |`offerId`| string | 2,048 characters | ✔ | A unique identifier that includes your Publisher ID and Offer ID, which you can find in [Partner Center](https://partner.microsoft.com/dashboard). You must format the string as `publisherId.offerId`.|
 
+## meetingExtensionDefinition
+
+**Optional** - object
+
+Specify meeting extension definition. For more information, see [custom Together Mode scenes in Teams](../../apps-in-teams-meetings/teams-together-mode.md).
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`scenes`|array of objects| 5 items||Meeting supported scenes.|
+
+### meetingExtensionDefinition.scenes
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`id`|||✔| The unique identifier for the scene. This id must be a GUID. |
+|`name`| string | 128 characters |✔| The name of the scene. |
+|`file`|||✔| The relative file path to the scenes' metadata json file. |
+|`preview`|||✔| The relative file path to the scenes' PNG preview icon. |
+|`maxAudience`| integer | 50  |✔| The maximum number of audiences supported in the scene. |
+|`seatsReservedForOrganizersOrPresenters`| integer | 50 |✔| The number of seats reserved for organizers or presenters.|
+
+## authorization
+
+**Optional** — object
+
+Specify and consolidate authorization related information for the app.
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`permissions`||||List of permissions that the app needs to function.|
+
+### authorization.permissions
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`resourceSpecific`| array of objects|16 items||Permissions that guard data access on resource instance level.|
+
+### authorization.permissions.resourceSpecific
+
+|Name| Type|Maximum size|Required |Description|
+|---|---|---|---|---|
+|`type`|string||✔| The type of the resource-specific permission. Options: `Application` and `Delegated`.|
+|`name`|string|128 characters|✔|The name of the resource-specific permission. <br> For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
+
+### Delegated permissions
+
```

Comment:
Please add a line.

Delegated permissions allow the app to access data on behalf of the signed-in user.